src/BankSystem/Account.java

```diff
@@ -1,13 +0,0 @@
-package BankSystem;
-
-public class Account extends BankAccount{
-    private String accountType;
```

Comment:
The accountType field is declared but not used anywhere in the code. I recommend removing it or utilizing it in the appropriate methods to ensure its purpose and relevance in the class.